tests/html/service-worker/index.html

```diff
@@ -1,35 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <title>Service Worker Tests</title>
 </head>
+
 <body>
-<p>This page tests the registration and initialization of Service Workers in Servo.</p>
+    <p>This page tests the registration and initialization of Service Workers in Servo.</p>
 
-<div id="sw_demo">
-    <iframe id="iframe_sw" width="" height="200"></iframe>
-    <br>
-    <a href="/dashboard-page.html">Go to Dashboard</a>
-    <a href="/profile.html">Go to Profile</a>
-    <br>
-    <button onclick="register_zero()">Register for Root (/)</button>
-    <button onclick="register_one()">Register for Dashboard (/dashboard)</button>
-    <button onclick="register_two()">Register for Profile (/profile)</button>
-    <button onclick="post_message_root()">Msg to Root service worker</button>
-</div>
+    <div id="sw_demo">
+        <iframe id="iframe_sw" width="" height="200"></iframe>
+        <br>
+        <a href="/dashboard-page.html">Go to Dashboard</a>
+        <a href="/profile.html">Go to Profile</a>
+        <br>
+        <button onclick="register_zero()">Register for Root (/)</button>
+        <button onclick="register_one()">Register for Dashboard (/dashboard)</button>
+        <button onclick="register_two()">Register for Profile (/profile)</button>
+        <button onclick="post_messaget()">Msg to Root service worker</button>
+    </div>
     <script>
-
-    var reg = navigator.serviceWorker.register('iframe_sw.js', { 'scope': '/demo_iframe.html' });
-    console.log("From client worker registered: ", navigator.serviceWorker.controller);
-    console.log("Registered script source: "+ navigator.serviceWorker.controller.scriptURL);
-    document.getElementById('iframe_sw').src = 'demo_iframe.html';
-    post_message('/demo_iframe');
+        navigator.serviceWorker.register('iframe_sw.js', { 'scope': '/demo_iframe.html'})
+        .then(function(registration) {
+            console.log("From client worker registered: ", navigator.serviceWorker.controller + "\nRegistered script source: " + "iframe_sw.js");
```

Comment:
When I said combine into one string, I didn't mean combining two console.log invocations into one. I meant the pattern of `console.log("a literal string" + "another literal string")` can become `console.log("a literal string another literal string")` instead.